src/main/convert.js

```diff
@@ -112,6 +114,20 @@ export class ThriftFileConverter {
       )
       .join('\n')}|}`;
 
+  generateUnion = ({id: {name}, fields}: Struct) =>
+    `export type ${this.transformName(name)} = ${this.generateUnionContents(fields)};`;
+
+  generateUnionContents = (fields: Object) => {
+    if (!fields.length) {
+      return this.generateStructContents(fields);
```

Comment:
I'd prefer you just use `{||}` here rather than calling generateStructContents with an empty array